pkg/apis/cstor/v1/cstorpoolinstance.go

```diff
@@ -118,12 +118,27 @@ type CStorPoolInstanceCapacity struct {
 	Used resource.Quantity `json:"used"`
 }
 
-type CSPIConditionType string
+type CStorPoolInstanceConditionType string
+
+const (
+	// CSPIPoolExpansion condition will be available when user triggers
+	// pool expansion by adding blockdevice/raidgroup (or) when underlying
+	// disk got expanded
+	CSPIPoolExpansion CStorPoolInstanceConditionType = "PoolExpansionInProgress"
+	// CSPIDiskReplacement condition will be available when user triggers
+	// disk replacement by replacing the blockdevice
+	CSPIDiskReplacement CStorPoolInstanceConditionType = "DiskReplacement"
```

Comment:
Can we have `DiskReplacementInProgess` to make conditions similar?

Reply:
This continuous tense will be represented in reason instead of  condition.